### DISCUSSION TITLE
Making the Background Image of the Home Page Transparent

### DISCUSSION THREAD

**Discussion by john995x** [hugo-themes/toha#318] [Q&A]
Hello,

Please can you tell me how I might be able to make the background of the site transparent and not blurred, like it comes after the initial set-up of the site?

I am studying to become a computer systems engineer, but don't have a whole lot of experience coding and have been getting a little frustrated trying to figure out how to remove the blur that is put a top the background image. 

Here is an example that I found where the blur has been removed and the background image is viewable. 

I read that this effect is achieved with some CSS code, but after swapping in and out new CSS files and trying to modify the values that correspond to opacity or blur myself with no luck, I figured I'd make a post as you guys who put this theme together are likely to know a solution. 

Thanks! 

Site for reference ---> https://jgabriel.eu/

**Comment by hossainemruz** [comment]
Hi @john995x create a `style.css` file in `static/css` directory of your repo. Then add the following css there.
```yaml
.background{
    filter: none!important;
    opacity: 0.9;
}
```

**Reply by hangangmagazine** [reply]
I did as instructed and it doesn't work for me.

Should this be added in a new `style.css` or added to the `/theme/toha/static/css/style.css `one?

**Reply by hossainemruz** [reply]
Actually, the issue was with the theme. The user style wasn't imported properly. I have fixed the issue and released version `v3.0.1` for the theme.

I have sent you a pull request that uses the latest release and add a custom css file for background transparency.

**Reply by hossainemruz** [reply]
@john995x can you please try as it has been done in this PR: https://github.com/hangangmagazine/hangangmagazine.github.io/pull/1

Make sure to update the theme to `v3.0.1`.
